presignedurl_tests.php

```diff
@@ -35,18 +35,26 @@
 
 echo $output->header();
 echo $output->heading(get_string('presignedurl_testing:page', 'tool_objectfs'));
+$delete = optional_param('delete', 0, PARAM_INT);
+$settingslink = \html_writer::link(new \moodle_url('/admin/settings.php?section=tool_objectfs'),
+    get_string('presignedurl_testing:objectfssettings', 'tool_objectfs'));
 
 $config = manager::get_objectfs_config();
 $support = false;
 if (!empty($config->filesystem)) {
     $fs = new $config->filesystem();
     $support = $fs->supports_presigned_urls();
 }
-$settingslink = \html_writer::link(new \moodle_url('/admin/settings.php?section=tool_objectfs'),
-    get_string('presignedurl_testing:objectfssettings', 'tool_objectfs'));
-
 if ($support) {
-
+    if ($delete === 1) {
+        $output->delete_presignedurl_tests_files();
```

Comment:
You don't check the sesskey here. It's not critical in this case but it's a good habit to keep up